### ISSUE TITLE
[Bug]: LC0095 false positive when casing of interface and implementation method differs

### ISSUE THREAD

**Issue by MODUSCarstenScholling** [ALCops/Analyzers#380]
### Analyzer

LinterCop

### Issue type

False positive (diagnostic fires when it shouldn't)

### Rule ID

LC0095

### Description

LC0095 fires false positive when the casing of an interface method and implementation method differs.

### AL code to reproduce

```al
codeunit 50102 MyCodeunit implements MyInterface
{
    procedure Myprocedure(MyInteger: Integer; MyText: Text; MyDate: Date)
    begin
        MyText := 'Hello';
        MyDate := Today();
    end;
}

interface MyInterface
{
    procedure MyProcedure(MyInteger: Integer; MyText: Text; MyDate: Date);
}
```

### Additional context

Fix is ready and I will create a PR.
